localnet/init/scripts/start-deposit-monitor.sh

```
@@ -0,0 +1,29 @@
+source $LOCALNET_INIT_DIR/../helper.sh
+
+# On some machines (e.g. MacOS), 172.17.0.1 is not accessible from inside the container, so we need to use host.docker.internal
+if [[ $CI == true ]]; then
+  export CFDM_BROKER_API_URL='ws://172.17.0.1:10997'
+  export CFDM_CHAINFLIP_RPC_URL='ws://172.17.0.1:9944'
+else
+  export CFDM_BROKER_API_URL='ws://host.docker.internal:10997'
+  export CFDM_CHAINFLIP_RPC_URL='ws://host.docker.internal:9944'
+
+fi
+$DOCKER_COMPOSE_CMD -f $LOCALNET_INIT_DIR/../docker-compose.yml -p "chainflip-localnet" up $DEPOSIT_MONITOR_CONTAINER $additional_docker_compose_up_args -d \
```

Shellcheck (0.10.0)

[warning] additional_docker_compose_up_args is referenced but not assigned.

(SC2154)